README.md

```diff
@@ -16,12 +16,14 @@ Xdrgen is a very early project.  Aside from the test fixtures in
 are the .x files used for the
 [stellar-core project](https://github.com/stellar/stellar-core).
 
-Xdrgen presently supports three output languages:  ruby, javacript, java, and golang:
+Xdrgen presently supports these output languages:  ruby, javacript, java,
+golang, and elixir:
 
 - ruby: complete support
 - javascript: complete support
 - java: complete support
 - golang: currently using a fork of go-xdr, but has complete support
+- elixir: support is experimental as the SDK is in early development
```

Comment:
@grossvogel let's mention here https://github.com/revelrylabs/exdr  and that it is required in your deps.

Reply:
@abuiles Yes, I'll make that readme update for sure. Thank you for the reminder to implement `String.Chars` in the `exdr` library as well. It's something I've meant to do but kept forgetting. I created [an issue](https://github.com/revelrylabs/exdr/issues/17) and hope to get to it in the next couple of weeks.

Reply:
@grossvogel awesome,  I think we can merge this once we have the link to `exdr`

Reply:
OK, I added the note. Hopefully the wording is OK 😸

Reply:
awesome, thanks!